auto_nag/scripts/severity_inconsistency.py

```diff
@@ -0,0 +1,92 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+WHITEBOARD_PAT = re.compile(r"\][,; ]*?\[")
+
+
+class SeverityInconsistency(BzCleaner):
+    def description(self):
+        return "Bugs with inconsistent severity flags"
+
+    def has_needinfo(self):
+        return True
+
+    def handle_bug(self, bug, data):
+        flags = WHITEBOARD_PAT.split(bug["whiteboard"][1:-1])
+        whiteboard_severities = [flag for flag in flags if flag.startswith("access-s")]
+        assert len(whiteboard_severities) == 1
```

Comment:
nit: it should be possible to get the severity thanks to a group in the regexp.

Reply:
Thank you! Yes, I agree with you. Initially, this solution was supposed to find the first `access-s` match to use it in the needinfo and the email without using any regexp.

I applied your suggestion by using the regexp to get all matches.